datumaro/components/crypter.py

```diff
@@ -29,17 +39,17 @@ def encrypt(self, msg: bytes) -> bytes:
 
     def handshake(self, key: bytes) -> bool:
         try:
-            return self.decrypt(key) == self._key
+            return self.decrypt(key) == self.key
```

Comment:
is this `self.key` instead of just `key`?

Reply:
This is because `handshake(key)` tries to check the equality between `self.key` and `key` which comes from the encrypted bytes in the annotation file. Therefore, we have to decrypt the bytes, `key` first (`self.decrypt(key)`) and compare it with `self.key`. Because both should have the `datum-` prefix, I changed it from `self._key` (without prefix) to `self.key` (with prefix).